walkthroughs/howto-k8s-grpc/v1beta2/manifest.yaml.template

```diff
@@ -0,0 +1,193 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${APP_NAMESPACE}
+  labels:
+    mesh: ${MESH_NAME}
+    appmesh.k8s.aws/sidecarInjectorWebhook: enabled
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: Mesh
+metadata:
+  name: ${MESH_NAME}
+spec:
+  namespaceSelector:
+    matchLabels:
+      mesh: ${MESH_NAME}
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualNode
+metadata:
+  name: client
+  namespace: ${APP_NAMESPACE}
+spec:
+  podSelector:
+    matchLabels:
+      app: client
+  listeners:
+    - portMapping:
+        port: 8080
+        protocol: http
+  backends:
+    - virtualService:
+        virtualServiceRef:
+          name: color
+  serviceDiscovery:
+    awsCloudMap:
+      namespaceName: ${CLOUDMAP_NAMESPACE}
+      serviceName: client
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualNode
+metadata:
+  name: server
+  namespace: ${APP_NAMESPACE}
+spec:
+  podSelector:
+    matchLabels:
+      app: color
+      version: server
+  listeners:
+    - portMapping:
+        port: 8080
+        protocol: grpc
+      healthCheck:
+        port: 8080
+        protocol: grpc
+        path: '/ping'
+        healthyThreshold: 2
+        unhealthyThreshold: 3
+        timeoutMillis: 2000
+        intervalMillis: 5000
+  serviceDiscovery:
+    awsCloudMap:
+      namespaceName: ${CLOUDMAP_NAMESPACE}
+      serviceName: color
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualService
+metadata:
+  name: color
+  namespace: ${APP_NAMESPACE}
+spec:
+  awsName: color.${CLOUDMAP_NAMESPACE}
```

Comment:
seems the client is talking to "color.${APP_NAMESPACE}.svc.cluster.local:8080", i believe this awsName need to be that to work

Reply:
CLOUDMAP_NAMESPACE="${APP_NAMESPACE}.svc.cluster.local" - but I see it can be confusing so I'll change the variable